debezium-platform-conductor/src/main/java/io/debezium/platform/environment/operator/actions/DebeziumServerProxy.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.platform.environment.operator.actions;
+
+import java.util.Map;
+import java.util.Optional;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.ws.rs.core.Response;
+
+import org.eclipse.microprofile.rest.client.inject.RestClient;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.debezium.DebeziumException;
+import io.debezium.operator.api.model.DebeziumServer;
+import io.debezium.platform.data.dto.SignalRequest;
+import io.debezium.platform.environment.actions.client.DebeziumServerClient;
+import io.fabric8.kubernetes.api.model.Service;
+import io.fabric8.kubernetes.api.model.ServiceList;
+import io.fabric8.kubernetes.client.KubernetesClient;
+
+@ApplicationScoped
+public class DebeziumServerProxy {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(DebeziumServerProxy.class);
+
+    private static final String DEBEZIUM_IO_CLASSIFIER_LABEL = "debezium.io/classifier";
+    private static final String DEBEZIUM_IO_INSTANCE_LABEL = "debezium.io/instance";
+    private static final String API_CLASSIFIER = "api";
+    private static final String SERVICE_URL_FORMAT = "http://%s:%s";
+
+    private final KubernetesClient k8s;
+    private final DebeziumServerClient dsClient;
+
+    public DebeziumServerProxy(KubernetesClient k8s, @RestClient DebeziumServerClient dsClient) {
+        this.k8s = k8s;
+        this.dsClient = dsClient;
+    }
+
+    public void sendSignal(SignalRequest signalRequest, DebeziumServer ds) {
+
+        var baseUrl = getDSApiBaseUrl(ds);
+        if (baseUrl.isEmpty()) {
+            throw new DebeziumException("Unable to find pipeline instance to send the signal");
+        }
+
+        var dsApiBaseUrl = baseUrl.get();
+        try (Response response = dsClient.sendSignal(dsApiBaseUrl, signalRequest)) {
+
+            LOGGER.debug("Call to {} returned with {}", baseUrl, response.getStatusInfo().getReasonPhrase());
```

Comment:
I noticed that here, it's logging the `Optional` instead the value. WDYT to pass the value?